docs/configuration/examples.md

```diff
@@ -431,6 +431,30 @@ jobs:
 
 </details>
 
+<details>
+  <summary>Extending timeout of pipeline for 24 hours (default is 12 hours)</summary>
```

Comment:
```suggestion
  <summary>Extending timeout of a test execution to 24 hours (default is 12 hours)</summary>
```

just so that it doesn't clash with Packit's pipeline terminology (see https://dashboard.packit.dev/pipelines)

Reply:
Yeah sure 💯 Thanks :)